.travis.yml

```diff
@@ -116,7 +116,16 @@ before_install:
       sudo apt-get update;
       sudo apt-get install sagemath-upstream-binary;
     fi
+#The install cycle below is to test installation on systems without setuptools.
```

Comment:
Move this below `install` if possible. Either above or below the line `- if [[ "${TEST_AUTOWRAP}" == "true" ]]; then`.